### PR TITLE
add minIO import script

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: scripts/minIO_import.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import concurrent.futures
+import os
+from pathlib import Path
+
+import boto3
+from tqdm import tqdm
```
Comment (MrCreosote) [inline scripts/minIO_import.py:6]: Can you add a requirements.txt with pinned versions? Pipenv would be better but not necessary

File: scripts/minIO_import.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+"""
+This script uploads genome files from collections NCBI source directory to the specified S3 bucket.
+
+Before running this script, make sure the following steps are completed:
+1. Start the SSH tunnel to the minIO server at NERSC:
```
Comment (MrCreosote) [inline scripts/minIO_import.py]:
```suggestion
1. Start the SSH tunnel to the minIO server at KBase Berkeley:
```

File: scripts/minIO_import.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    ssh -L 9002:ci07:9002 $USERNAME@login1.berkeley.kbase.us -N
+2. Get the secret key from the minIO server in Rancher and set the environment variable:
+    i. Go to the minIO server in Rancher
+        Stacks → CI → cdm-minio -> ci-core-cdm-minio-1 → Execute Shell → echo $MINIO_ROOT_PASSWORD
```
Comment (MrCreosote) [inline scripts/minIO_import.py:16]: You can also get it from the Command tab without having to execute a shell

File: scripts/minIO_import.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+            # skip uploading if the file already exists in the bucket
+            s3.head_object(Bucket=bucket, Key=s3_key)
+        except s3.exceptions.ClientError:
+            s3_client.upload_file(upload_file, bucket, s3_key)
+    else:
+        multi_match_genome_ids.append(genome_id) if matching_files else no_match_genome_ids.append(genome_id)
+
+
+if __name__ == '__main__':
+    meta_dir = Path('/global/homes/t/tgu/GTDB_meta')
```
Comment (MrCreosote) [inline scripts/minIO_import.py]: If you wind up making changes it'd be better if:
```suggestion
def main():
   ...

if __name__ == '__main__':
    main()
```
So all these variable names aren't globals. Not necessary for this PR though

File: scripts/minIO_import.py [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+                                   SUFFIX,
+                                   BUCKET,
+                                   multi_match_genome_ids,
+                                   no_match_genome_ids) for genome_id in lineage_genome_ids]
+
+        for future in concurrent.futures.as_completed(futures):
+            future.result()
+            progress_bar.update(1)
+
+    progress_bar.close()
```
Comment (MrCreosote) [inline scripts/minIO_import.py:126]: nice

File: scripts/minIO_import.py (hunk 1 above)
Comment (Tianhao-Gu) [reply scripts/minIO_import.py:6]: 👍

File: scripts/minIO_import.py (hunk 4 above)
Comment (Tianhao-Gu) [reply scripts/minIO_import.py]: 👍

File: scripts/minIO_import.py (hunk 3 above)
Comment (Tianhao-Gu) [reply scripts/minIO_import.py:16]: 👍 Didn't know it before.

File: scripts/minIO_import.py (hunk 2 above)
Comment (MrCreosote) [reply scripts/minIO_import.py]: :+1:

File: scripts/minIO_import.py (hunk 4 above)
Comment (MrCreosote) [reply scripts/minIO_import.py]: :+1:

File: scripts/minIO_import.py (hunk 1 above)
Comment (MrCreosote) [reply scripts/minIO_import.py:6]: :+1:

File: scripts/minIO_import.py (hunk 3 above)
Comment (MrCreosote) [reply scripts/minIO_import.py:16]: :+1:

**Review by MrCreosote — APPROVED** [review]

**Pull request merged**